SensusService/Protocol.cs

```diff
@@ -1012,6 +1121,50 @@ public void Start()
             }
         }
 
+        public void ScheduleStart()
+        {
+            DateTime startTime = new DateTime(_startTime.Year, _startTime.Month, _startTime.Day, _startTime.Hour, _startTime.Minute, 0);
```

Comment:
We definitely could, it will just include seconds in the scheduled time. I did this to set seconds to 0. I could use this (http://stackoverflow.com/a/1859266) sort of thing if that would be better, or just let it take the seconds into account?